<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 66px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 71px;
  margin-top: -71px;
}

.section h2 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h3 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h4 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h5 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h6 {
  padding-top: 71px;
  margin-top: -71px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="projects.html">Research projects</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="teaching.html">Teaching</a>
</li>
<li>
  <a href="awards.html">Awards and services</a>
</li>
<li>
  <a href="photos.html">Photos</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/shankarkshakya">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/shakya">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/shankar-kaji-shakya-30204455/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="ph.d.research-at-oregon-state-university" class="section level1">
<h1><strong>Ph.D.Â research at Oregon State University</strong></h1>
<p>My Ph.D dissertation work at OSU has focused primarily on oomycete pathogen <em>Phytophthora</em>. This includes population genetics analyses of <em>P. infestans</em> at its center of origin using microsatellite markers to using whole genomes to understand the evolution of effector genes in clade 1c species of genus <em>Phytophthora</em>.</p>
<div id="analysis-of-p.-infestans-population-from-mexico-using-microsatellite-markers" class="section level2">
<h2>1. <strong>Analysis of <em>P. infestans</em> population from Mexico using microsatellite markers</strong></h2>
<p>Microsatellites or simple sequence repreats (SSR) is probably the mostly widely used marker in any population genetic analyses however this is changing as the cost of sequencing has down. We performed analyses to describe the gradient in diversity as we move away from the center of diversity.</p>
<p><img src="mexico_paper/Fig1.png" style="width:50%; border:3px solid; margin-left: 80px" align="center"></p>
<p><img src="mexico_paper/Fig3.png" style="width:50%; border:3px solid; margin-left: 80px" align="center"></p>
<p>This work has already been published and can be found <a href="https://apsjournals.apsnet.org/doi/10.1094/PDIS-11-17-1801-RE?mobileUi=0">here</a>.</p>
</div>
<div id="potential-center-of-origin-of-phytophthora-cinnamomi" class="section level2">
<h2>2. <strong>Potential center of origin of <em>Phytophthora cinnamomi</em></strong></h2>
<p><em>P. cinnamomi</em> is a globally distributed, broad host range, soil borne oomycete pathogen. The center of origin of the pathogen is still a debate. We aim at using genome wide SNP data to analyse the global population of <em>P. cinnamomi</em> to infer its potential center of origin. Our analyses demonstrates presence of two distinct clonal lineages of A2 mating types and rare occurence of A1 mating type except in Asia. Isolates from Taiwan and Vietanam showed increased diversity compare to other geopgraphic regions.</p>
<p><img src="mexico_paper/pcinna.png" style="width:70%; border:3px solid; margin-left: 80px" align="center"></p>
<p>This work was presented at ICPP 2018 meeting in Boston and the poster can be found <a href="https://www.researchgate.net/publication/326816966_Geographic_origin_of_the_jarrah_dieback_pathogen_Phytophthora_cinnamomi">here</a>.</p>
</div>
<div id="evolution-of-rxlr-effectors-in-three-phytophthora-clade-1c-species" class="section level2">
<h2>3. <strong>Evolution of RXLR effectors in three <em>Phytophthora</em> clade 1c species</strong></h2>
<p>Successful pathogens have evolved effectors to suppress the basal immunity and promote virulence. One class of such effectors in genus <em>Phytophthora</em> is RXLR effectors. These effectors are recognized by RXLR amino acid domain (Arginine-any AA- Leucine-Arginine). There are hundredrs of RXLR effectors in <em>Phytophthora</em> and these profile varies depending on species and strain. Here we use whole genome sequences to infer the core RXLR effectors in three closely related <em>Phytophthora</em> species namely <em>P. infestans, P. ipomoeae and P. mirabilis</em>.</p>
<p><img src="mexico_paper/boc.png" style="width:70%; border:3px solid; margin-left: 80px" align="center"></p>
<p>This work was presented at annual phytopathological society meeting 2018 in San Antonio, TX. Abstract can be found <a href="https://apsnet.confex.com/apsnet/2017/meetingapp.cgi/Paper/5959">here</a>.</p>
</div>
<div id="assembly-of-unmapped-reads" class="section level2">
<h2>4. <strong>Assembly of unmapped reads</strong></h2>
<p>Most of the resequencing projects involve mapping short reads to reference genome. However some percetage of reads do not map in the process and these reads are often neglected. Our objective here is to assemble the unmapped reads and make gene calls. These genes are probably unique to the strain.</p>
<hr />
</div>
</div>
<div id="m.s.-research-at-university-of-florida" class="section level1">
<h1><strong>M.S. research at University of Florida</strong></h1>
<p>My M.S. work at University of Florida was focused on understanding the epidemioloical aspect of late blight disease. Most the controlled epidemiological studies use constant temperature to describe the development of the disease however in natural setting temperature keeps changing. We wanted to understand the differences in disease development at constant and fluctuating temperature conditions.</p>
<p>This work has already been published and can be found <a href="https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-05-14-0132-R">here</a>.</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
